Mage/src/main/java/mage/MageObject.java

```diff
@@ -283,6 +285,12 @@ default boolean isWorld(Game game) {
         return getSuperType(game).contains(SuperType.WORLD);
     }
 
+    default boolean canTurnFaceDown(Game game) {
+        return !game.getPermanent(getId()).isFaceDown(game)
```

Comment:
why is this a method in MageObject rather than in Permanent? If it's here, then need null check on the permanent.

Reply:
Yes, my original comment was about overrides by permanent and card implementations (e.g. mdf cards must return false, PermanentImpl must use current check, etc). It can be added to `Card.java` near `isFaceDown` and other related code.